lib/msal-common/src/authority/IOpenIdConfigResponse.ts

```diff
@@ -6,7 +6,7 @@
 /**
  * Tenant Discovery Response which contains the relevant OAuth endpoints and data needed for authentication and authorization.
  */
-export interface TenantDiscoveryResponse {
+export interface IOpenIdConfigResponse {
```

Comment:
Can we take off the `I` in front, just name this `OpenIdConfigResponse` and make this a type?